manifest.json

```diff
@@ -3,8 +3,13 @@
     "name": "Casey's Extension",
     "version": "1.0.0",
     "description": "Hello World!",
-    "permissions": ["activeTab"],
-    
+    "permissions": ["webRequest" , "http://*/*", "https://*/*", "webRequestBlocking"],
```

Comment:
When we have multiple array items like this, formatting best practice is to have each item on its own line. Check out our `manifest` file in the extension for an example of how the formatting should look.